fiftyone/operators/store/service.py

```
@@ -14,16 +15,26 @@
 
 
 class ExecutionStoreService:
-    """Service for managing execution store operations."""
+    def __init__(
+        self,
+        repo: Optional["ExecutionStoreRepo"] = None,
```

Ruff

Undefined name `ExecutionStoreRepo`

(F821)